### ISSUE TITLE
Manager crashed when quickly scrolling through commits

### ISSUE THREAD

**Issue by Wojtaz0w** [vendetta-mod/VendettaManager#10]
It just crashed after quickly scrolling through commits list (it also lagged before crashing) 
https://paste.crdroid.net/YHZgeg

**Comment by XAlboX** [comment]
Does this happen to you every time?

**Comment by XAlboX** [comment]
Does this happen to you every time?

**Comment by Wojtaz0w** [comment]
Yes happens every time,  heres the video https://youtube.com/shorts/9e--sp2VUlk

**Comment by XAlboX** [comment]
I think it's a ROM problem :/

**Comment by mrrobboss** [comment]
Able to recreate on my Realme 8i. I noticed that when i dont record, i need to be really fast for it, but once i start recording it gets a lot easier to reproduce.

**Comment by Wojtaz0w** [comment]
My redmi 9 has 3gb,it may be a memory leak since it crashes easily

**Comment by wingio** [comment]
its a pretty simple issue for me to fix, for now just don't scroll so fast
